components/Input/Input.jsx

```diff
@@ -106,7 +106,7 @@ class Input extends React.Component {
       currentValue: inputValue,
     });
 
-    onChange();
+    onChange(ev, { inputValue });
```

Comment:
You really need the second parameter ? Because he is already inside the `ev` variable.

Reply:
No, I don't. 
I just keep it for consistency with the previous version that had it, since there might be other projects depending on this second param. But, IMO, he can be disposed.

If you guys agree with the absence of the second parameter I can remove it and update the PR.